packages/fuels-code-gen/src/program_bindings/abigen/bindings/function_generator.rs

```diff
@@ -38,7 +38,7 @@ impl FunctionGenerator {
         })
     }
 
-    pub fn set_name(&mut self, name: String) -> &mut Self {
+    pub fn with_name(&mut self, name: String) -> &mut Self {
```

Comment:
This seems to be a mix of what we try to separate:  `set_(&mut self) -> ()` and `with(self) -> Self`.
IMO I would not use `with` here as it is not consistent with the rest of our code and could be confusing.

Here are two have 2 options that come to mind: 
- use a different name (not `set` and not  `with` ). Maybe only the field name like `name`.
- use either `set` or  `with` and change the code to fit this

Reply:
Changed it to `name`. 
I figured it's too much of an impact since it's not a public facing API.

Reply:
Decided to leave it at  `set_` since there is some getters that occupy the `name` convention already.